### ISSUE TITLE
Metric finetuning -- error increases while training

### ISSUE THREAD

**Issue by tmagcaya** [DepthAnything/Depth-Anything-V2#183]
I'm following the instructions for metric finetuning. Started with a base VITS model then finetuned with vKITTI. Eval pref on real KITTI decreases over time.. Did anyone else have this issue?

Epoch 1:
==========================================================================================
[2024-10-13 07:37:37,613][    INFO]       d1,       d2,       d3,  abs_rel,   sq_rel,     rmse, rmse_log,    log10,    silog
[2024-10-13 07:37:37,614][    INFO]    0.861,    0.970,    0.992,    0.120,    0.626,    3.922,    0.161,    0.049,    0.152
[2024-10-13 07:37:37,614][    INFO] ==========================================================================================

Epoch 60:
 ==========================================================================================
[2024-10-13 18:56:12,798][    INFO]       d1,       d2,       d3,  abs_rel,   sq_rel,     rmse, rmse_log,    log10,    silog
[2024-10-13 18:56:12,798][    INFO]    0.789,    0.942,    0.982,    0.152,    0.935,    4.759,    0.202,    0.063,    0.186
[2024-10-13 18:56:12,798][    INFO] ==========================================================================================

Epoch 120:
 ==========================================================================================
[2024-10-14 06:02:20,092][    INFO]       d1,       d2,       d3,  abs_rel,   sq_rel,     rmse, rmse_log,    log10,    silog
[2024-10-14 06:02:20,093][    INFO]    0.786,    0.935,    0.978,    0.150,    0.975,    4.932,    0.208,    0.063,    0.193
[2024-10-14 06:02:20,093][    INFO] ==========================================================================================

[20241013_072607.log](https://github.com/user-attachments/files/17359737/20241013_072607.log)

Full logs below

Dist Train:
#!/bin/bash
now=$(date +"%Y%m%d_%H%M%S")

epoch=120
bs=8
gpus=1
lr=0.000005 #0.000005 batch size 4
encoder=vits
dataset=vkitti
img_size=518
min_depth=0.001
max_depth=80 # 80 for virtual kitti
pretrained_from=../checkpoints/depth_anything_v2_${encoder}.pth
save_path=exp/vkitti

mkdir -p $save_path

python3 -m torch.distributed.launch \
    --nproc_per_node=$gpus \
    --nnodes 1 \
    --node_rank=0 \
    --master_addr=localhost \
    --master_port=20596 \
    train.py --epoch $epoch --encoder $encoder --bs $bs --lr $lr --save-path $save_path --dataset $dataset \
    --img-size $img_size --min-depth $min_depth --max-depth $max_depth --pretrained-from $pretrained_from \
    --port 20596 2>&1 | tee -a $save_path/$now.log

**Comment by vineyard0531** [comment]
@tmagcaya I also encountered the same problem. Have you solved your problem?

**Comment by tmagcaya** [comment]
@vineyard0531 Nope, I found many other issues in the pull request tab of this repo that are somewhat relevant. I think authors left out some key pieces from this repo as they work on V3... Thanks for wasting open-source community time
